vars/main.yml

```diff
@@ -19,3 +20,25 @@ cis_audit_auditd_filename:  "/etc/audit/auditd.conf"
 cis_audit_rules_filename: "/etc/audit/audit.rules"
 
 full_upgrade: True
+
+fs_mounts:
+  - mountpoint: '/tmp'
+    device: 'tmpfs'
+    fstype: 'tmpfs'
+    opts: 'rw,nosuid,nodev,noexec,relatime'
+  - mountpoint: '/var/tmp'
+    device: 'tmpfs'
+    fstype: 'tmpfs'
+    opts: 'rw,nosuid,nodev,noexec,relatime'
+  - mountpoint: '/dev/shm'
+    device: 'tmpfs'
+    fstype: 'tmpfs'
+    opts: 'rw,nosuid,nodev,noexec,relatime'
+
+cis_level_1_exclusions:
```

Comment:
I don't think we should put these here either. They should only go into the playbook that uses the role.